bin/comment_on_pr

```diff
@@ -1,14 +1,125 @@
 #!/bin/bash -eu
 
-PR_MESSAGE_FILE=$1
-GITHUB_TOKEN=$2
```

Comment:
I used lowercase for the variable names. Using upper case, plus the fact that there are lots of variable declared in this script, makes me feel like the whole script is screaming.